IDE/IAR-EWARM/Projects/user_settings.h

```diff
@@ -17,10 +16,59 @@
 
 #define WOLFSSL_GENSEED_FORTEST /* Warning: define your own seed gen */
 
-#define TFM_TIMING_RESISTANT
+/* A few examples of different math options below.
+ *
+ * See examples/configs/user_settings_template.h for a more
+ * detailed template. */
+#if 1
```

Comment:
This section for single precision can exist all on its own. It can be used regardless of the 3 multi-precision math library options as long as `WOLFSSL_SP_MATH` is not defined.